docs/getting_started/execution-modes.rst

```diff
@@ -184,3 +184,33 @@ Each task will create a new container on Azure, giving full isolation. This, how
             "image": "dbt-jaffle-shop:1.0.0",
         },
     )
+
+
+.. _invocation_modes:
+Invocation Modes
+================
+.. versionadded:: 1.4
+
+For ``ExecutionMode.LOCAL`` execution mode, Cosmos supports two invocation modes for running dbt:
+
+1. ``InvocationMode.SUBPROCESS``: In this mode, Cosmos runs dbt cli commands using the Python ``subprocess`` module and parses the output to capture logs and to raise exceptions.
+
+2. ``InvocationMode.DBT_RUNNER``: In this mode, Cosmos uses the ``dbtRunner`` available for `dbt programmatic invocations <https://docs.getdbt.com/reference/programmatic-invocations>`__ to run dbt commands. \
+   In order to use this mode, dbt must be installed in the same environment, either local or virtualenv for the worker. This mode does not have the overhead of spawning new subprocesses or parsing the output of dbt commands and is faster than ``InvocationMode.SUBPROCESS``. \
+   This mode requires dbt version 1.5.0 or higher.
```

Comment:
It is probably worth linking that it is up to the user to resolve any dependencies conflicts between Airflow an dbt, and add a link to: https://astronomer.github.io/astronomer-cosmos/getting_started/execution-modes-local-conflicts.html#execution-modes-local-conflicts